src/containers/other/NFTView/hooks/useNftData.ts

```diff
@@ -12,6 +12,7 @@ import { useCurrentDomain } from 'lib/providers/CurrentDomainProvider';
 import useNotification from 'lib/hooks/useNotification';
```

Comment:
This hook is getting pretty heavy - definitely need to split it out into grouped functionality at some point, but maybe outside the scope of this task!